src/messages/javascript.js

```diff
@@ -0,0 +1,15 @@
+import { gettext } from 'utils';
+
+
+// These constants aren't fullcaps so we can use filenames/rule names/
+// message names consistently across the app.
+export const mozIndexedDB = {
+  code: 'mozIndexedDB',
+  message: gettext('mozIndexedDB has been removed; use indexedDB instead.'),
+  description: gettext('mozIndexedDB has been removed; use indexedDB instead.'),
+};
+export const mozIndexedDB_prop = {
+  code: 'mozIndexedDB_prop',
```

Comment:
We can do that. I actually think me doing it this way is a holdover from something I hacked together and never removed.

I'll aim to make these more CONSTANT_LIKE.